extension/package.json

```diff
@@ -10,10 +10,10 @@
     "prod": {}
   },
   "scripts": {
-    "clean": "rm -rf dist .parcel-cache",
+    "clean": "rmdir dist .parcel-cache",
     "cleanbuild": "yarn run clean && yarn run build",
-    "copy": "mkdir dist && cp manifest.json dist/",
-    "build": "yarn run copy && parcel build src/*"
+    "copy": "mkdir dist && copy manifest.json dist\\",
+    "build": "yarn run copy && parcel build src\\*"
```

Comment:
We should use environment variables to determine the platform and change the commands or just add them as separate ones, e.g. cleanwin or copywin